pages/common/gifsicle.md

```diff
@@ -1,20 +1,32 @@
 # gifsicle
 
-> Create GIFs.
+> GIF manipulator.
 > More information: <https://www.lcdf.org/gifsicle>.
 
-- Optimise a GIF:
+- Optimize a GIF as a new file:
 
-`gifsicle --batch --optimize=3 {{amin.gif}}`
+`gifsicle {{path/to/input.gif}} --optimize=3 -o {{path/to/output.gif}}`
 
-- Make a GIF animation with gifsicle:
+- Unoptimize a GIF in place:
 
-`gifsicle --delay={{10}} --loop *.gif > {{anim.gif}}`
+`gifsicle -b {{path/to/input.gif}} --unoptimize`
 
-- Extract frames from an animation:
+- Extract a frame from an animation:
 
-`gifsicle {{anim.gif}} '#0' > {{firstframe.gif}}`
+`gifsicle {{path/to/input.gif}} '#{{0}}' > {{path/to/firstframe.gif}}`
 
-- You can also edit animations by replacing, deleting, or inserting frames:
+- Make a GIF animation from selected still/animated GIFs:
 
-`gifsicle -b {{anim.gif}} --replace '#0' {{new.gif}}`
+`gifsicle {{*.gif}} --delay={{10}} --loop > {{path/to/input.gif}}`
+
+- Reduce file size using multiple methods at the expense of quality:
```

Comment:
```suggestion
- Reduce file size using compression:
```

Reply:
That's fair, although I don't really think of reducing the number of colors as traditional compression. What do you think of "multiple compression methods". Or maybe "several" instead of "multiple".

Reply:
On second thought, I think this is good.